README.md

```diff
@@ -240,12 +240,12 @@ All properties are mandatory.
 }
 ```
 
-Only the `topic` and properties are mandatory
+Only the `topic` and `payload` properties are mandatory.
 Both `topic` and `payload` can be `Buffer` objects instead of strings.
 `messageId` is mandatory for `qos > 0`.
 
 If `payload` is passed to `generate(packet)` as a string, it will be
-automatically converted into a `Buffer`.
+automatically converted to a `Buffer`.
```

Comment:
This might not be true. If writeToStream is used, the payload is written as a string if it is a string.